src/SFA.DAS.EmployerIncentives/Models/ClawbackStatus.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace SFA.DAS.EmployerIncentives.Models
+{
+    public class ClawbackStatus
+    {
+        public decimal? ClawbackAmount { get; set; }
+        public DateTime? ClawbackDate { get; set; }
+    }
```

Comment:
in what context are these going to be `null`?
also Clawback prefix is implied and not needed imho 

```suggestion
        public decimal Amount { get; set; }
        public DateTime Date { get; set; }
```

Reply:
ah the prop name prefix is consistent with PaymentStatus so i guess is ok